network-api/networkapi/wagtailpages/pagemodels/publications/publication.py

```diff
@@ -72,20 +95,16 @@ class PublicationPage(FoundationMetadataPageMixin, Page):
     )
 
     content_panels = Page.content_panels + [
-        FieldPanel('subtitle'),
-        FieldPanel('secondary_subtitle'),
-        FieldPanel('publication_date'),
-        ImageChooserPanel('hero_image'),
-        DocumentChooserPanel('publication_file'),
+        MultiFieldPanel([
+            InlinePanel("authors", label="Author", min_num=1)
```

Comment:
Authors should be totally optional, `min_num=None` would be more appropriate and prevent editors/authors from getting confused about a required field if it isn't required.